docker/scripts/extract_thumbnail.sh

```diff
@@ -43,6 +43,11 @@ elif [[ "$MIMETYPE" =~ ^video.* ]]; then
     mv /tmp/mediafile /tmp/videofile
     extract_video_thumbnail.sh "$1" "$2" "$3" "$4"
     exit $?
+elif [[ "$MIMETYPE" =~ ^image/x-canon-cr2.* ]] || [[ "$MIMETYPE" =~ ^image/tiff.* ]]; then
```

Comment:
hmm, please could you check this against a "normal" TIFF image and check that doesn't break?

Reply:
Those did break, but I have fixed this.